### ISSUE TITLE
git diff terminal colors not working

### ISSUE THREAD

**Issue by fllaca** [hassio-addons/addon-ssh#30]
# Problem/Motivation

> (Why the issue was filed)

As stated in https://github.com/home-assistant/hassio/issues/343, `git diff` shows bad colouring (In my case when using a git repo to store homeassistant configuration).

## Expected behavior

> (What you expected to happen)

Git diff shows output with nice colours:
![imagen](https://user-images.githubusercontent.com/14851307/43685503-849f306a-98b4-11e8-941e-c8332494cb60.png)

## Actual behavior

> (What actually happened)
Git output shows uggly codes:

![imagen](https://user-images.githubusercontent.com/14851307/43685494-5e394bcc-98b4-11e8-94d8-9ab13eb759b7.png)

## Steps to reproduce

> (How can someone else make/see it happen)
Use git repo to store configuration:
```
# ssh into hassio:
cd /config
git init
git add .
git commit -m "initial commit"
```
Then, make some changes:
```
cd /config
# change something:
vi configuration.yaml 

# This shows bad colours:
git diff
```

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

Following @jkeljo advice in https://github.com/home-assistant/hassio/issues/343,  installing a more feature version of less fixes it: `apk --no-cache add less`

**Comment by fllaca** [comment]
I've made a PR with a fix for this: #31

**Comment by frenck** [comment]
Added in `v3.3.0`! 🎉
